### DISCUSSION TITLE
View Collection Concatenation of fields

### DISCUSSION THREAD

**Discussion by Carl-P4** [pocketbase/pocketbase#3636] [Q&A] [answered]
Would like to combine a few fields in a view collection into a single column, `FirstName || ' ' || LastName` for instance,

I've an SQL command that works in DB Browser, but not currently in a pocketbase view

Is this what task : 
https://github.com/pocketbase/pocketbase/issues/317

Will allow?

**Comment by ganigeorgiev** [comment, marked as answer]
Column expressions for the view query should be already supported. The caveat is that you need to provide an alias that will be used as field name. For example:

```sql
SELECT
    id,
    (firstName || ' ' || lastName) as fullName
FROM users
```

**Reply by Carl-P4** [reply]
Ahhh my query in DB Browser didnt have brackets, added those in for PB and now working! 

Thanks
